cmd/common.go

```diff
@@ -34,10 +34,12 @@ func createIdentity(ctx context.Context, idService identity.Service) (identity.C
 }
 
 func generateKeys(config config.Configuration) {
-	p2pPub, p2pPvt := config.GetSigningKeyPair()
+	p2pPub, p2pPvt := config.GetP2PKeyPair()
+	signPub, signPvt := config.GetSigningKeyPair()
 	ethAuthPub, ethAuthPvt := config.GetEthAuthKeyPair()
-	crypto.GenerateSigningKeyPair(p2pPub, p2pPvt, "ed25519")
-	crypto.GenerateSigningKeyPair(ethAuthPub, ethAuthPvt, "secp256k1")
+	crypto.GenerateCryptoKeyPair(p2pPub, p2pPvt, "ed25519")
```

Comment:
Actually the name signing key is accurate because these are signature keys. The algorithm secp256k1 and ed25519 are sig schemes (as opposed to encryption schemes).

I would not rename it or just get rid of "Crypto" -> "GenerateKeyPair"